packages/graphiql-react/src/theme.tsx

```diff
@@ -53,4 +63,23 @@ export function useTheme() {
   return useMemo(() => ({ theme, setTheme }), [theme, setTheme]);
 }
 
+type ThemeContextProviderProps = { children: ReactNode };
+
+export function ThemeContextProvider(props: ThemeContextProviderProps) {
+  const theme = useThemeInternal();
```

Comment:
Nit: We could just inline the contents from that hook here and avoid the extra function.